examples/testprograms/http/autokitteh.yaml

```diff
@@ -2,19 +2,19 @@ version: v1
 
 project:
   name: http
-  vars:
-    - name: MEOW
-      value: "meow1"
-    - name: WOOF
-      value: "woof"
   connections:
-    - name: http1
-      token: "meow"
+    - name: myhttp
       integration: "http"
   triggers:
     - event_type: get
-      connection: http1
+      connection: myhttp
       entrypoint: http.star:on_http_get
+      data:
+        pattern: "get"
```

Comment:
1. What does "data" actually mean/contain?
2. What is the valid/expected contents of "pattern"?

Reply:
documented